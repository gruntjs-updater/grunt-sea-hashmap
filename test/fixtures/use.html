<!DOCTYPE html>
<html>
   <head>
        <meta charset="UTF-8">
        <title>grunt-sea-hashmap test</title>
        <script src="/static/seajs/sea.js" data-main="page/c?hash"></script>
    </head>
    <body>

	</body>
</html>