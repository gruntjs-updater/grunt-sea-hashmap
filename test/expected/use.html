<!DOCTYPE html>
<html>
   <head>
        <meta charset="UTF-8">
        <title>grunt-sea-hashmap test</title>
        <script src="/static/seajs/sea.js" data-main="page/c?8c9bef5cb9b0891097f88c72accc48f3"></script>
    </head>
    <body>

	</body>
</html>